config/task_test.go

```diff
@@ -896,62 +920,3 @@ func TestTaskConfig_validateCondition(t *testing.T) {
 		})
 	}
 }
-
-func TestTaskConfig_FinalizeValidate(t *testing.T) {
-	// Tests the full finalize then validate process, particularly how it handles the
-	// relationship between task.services and task.condition
-	//
-	// Relationships:
-	// - When catalog-service condition's regexp is nil or empty string, test
-	//    how regexp is handled by Finalize() which impacts Validate()
-
-	cases := []struct {
-		name   string
-		config *TaskConfig
-		valid  bool
-	}{
-		{
-			"invalid: no services with catalog-service condition missing regexp",
-			&TaskConfig{
-				Name:      String("task_a"),
-				Module:    String("path"),
-				Condition: &CatalogServicesConditionConfig{},
-			},
-			false,
-		},
-		{
-			"valid: no services with catalog-service condition's regexp is empty string",
-			&TaskConfig{
-				Name:      String("task_a"),
-				Module:    String("path"),
-				Services:  []string{"serviceA"},
-				Condition: &CatalogServicesConditionConfig{CatalogServicesMonitorConfig{Regexp: String("")}},
-			},
-			true,
-		},
-		{
-			"valid: no module_input included with schedule condition",
```

Comment:
Removed this test case - no longer requiring any type of module input for schedule condition (or any condition for that matter). Feedback welcome!